### PR TITLE
adding the main.cpp

### PR DESCRIPTION
**Pull request by khachatryanani**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+long long factorial (int n)
+{
+	if(n==0 || n==1)
```
Comment (haykaslanian) [inline main.cpp]: For better readability add spaces after if,   add spaces after and before ==, *, -,...

File: main.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	{
```
Comment (haykaslanian) [inline main.cpp]: We usually use two or 4 spaces to indent code

File: main.cpp (hunk 1 above)
Comment (khachatryanani) [reply main.cpp]: Thank you for review. Code is changed, committed.

File: main.cpp (hunk 2 above)
Comment (khachatryanani) [reply main.cpp]: Thank you for the review, code is changed accordingly.

**Pull request merged**